http://bugzilla.gnome.org/show_bug.cgi?id=595907

--- origsrc/gtk-engines-2.18.3/engines/lua/Makefile.am	2009-09-06 12:12:22.000000000 -0500
+++ src/gtk-engines-2.18.3/engines/lua/Makefile.am	2009-09-22 01:37:36.520436100 -0500
@@ -22,11 +22,11 @@
 NULL =
 
 if SYSTEM_LUA
-	LIBLUA_SOURCES =
-	LIBLUA_CFLAGS = $(LUA_CFLAGS)
+LIBLUA_SOURCES =
+LIBLUA_CFLAGS = $(LUA_CFLAGS)
 else
-	LIBLUA_CFLAGS = -I$(top_srcdir)/engines/lua/src/liblua
-	LIBLUA_SOURCES = \
+LIBLUA_CFLAGS = -I$(top_srcdir)/engines/lua/src/liblua
+LIBLUA_SOURCES = \
 		./src/liblua/lapi.h \
 		./src/liblua/lauxlib.h \
 		./src/liblua/lcode.h \
