http://bugzilla.gnome.org/show_bug.cgi?id=594014

--- origsrc/gtk-engines-2.18.3/configure.ac	2009-09-21 14:49:18.000000000 -0500
+++ src/gtk-engines-2.18.3/configure.ac	2009-09-22 01:55:30.566109000 -0500
@@ -47,7 +47,7 @@ AC_ARG_ENABLE(deprecated,	[  --disable-d
 AC_ARG_ENABLE(schemas,		[  --disable-schema        disable engine schemas],	[schemas=$enableval],	[schemas="yes"])
 AC_ARG_WITH(system-lua,	[  --with-system-lua        link with system Lua library], 		[system_lua=$withval], 	[system_lua="no"])
 
-AM_CONDITIONAL([SYSTEM_LUA], [test x$system_lua = xtrue])
+AM_CONDITIONAL([SYSTEM_LUA], [test x$system_lua = xyes])
 
 BUILD_ENGINES=""
 BUILD_THEMES=""
